src/mapper.ts

```diff
@@ -169,7 +169,7 @@ export class ProgressItem {
   get formattedValue(): string {
     const value =
       this._panelConfig.getValue('valueLabelType') === 'percentage' ?
-      this.aggregatedProgress :
+      this.aggregatedProgress:
       this._aggregatedValue;
```

Comment:
Maybe like this?
```
const foo = maybe1 > maybe2
  ? 'bar'
  : maybeNull;
```

https://github.com/airbnb/javascript#comparison--nested-ternaries